kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/wires/PropertiesUi.java

```diff
@@ -1106,6 +1131,48 @@ public void onValueChange(final ValueChangeEvent<String> event) {
         this.fields.add(formGroup);
     }
 
+    private TextBoxBase createTextBox(final GwtConfigParameter param) {
+        if (param.getDescription() != null && param.getDescription().contains("\u200B\u200B\u200B\u200B\u200B")) {
```

Comment:
Yes, you're right.
Since we need to refactor this code to extend the AbstractServicesUI, I think it's better to track it in Issue #999 and do it once at that time.